source/import-export.txt

```diff
@@ -23,33 +23,68 @@ and export for both **JSON** and **CSV** files.
 Import Data into a Collection
 -----------------------------
 
-|compass| can import data into a collection from either a
-**JSON** or **CSV** file.
+|compass| can import data into a collection from either a **JSON** or
+**CSV** file.
 
-.. note::
+Format Your Data
+~~~~~~~~~~~~~~~~
 
-   When importing data from a **JSON** file, each document must
-   exist on its own line. There must also be a blank line at the end
-   of the file, or the last document will not be inserted.
+Before you can import your data into |compass| you must first ensure
+that it is formatted correctly.
 
-   The following example ``.json`` file imports three documents:
+.. tabs::
 
-   .. cssclass:: copyable-code
-   .. code-block:: javascript
+   tabs:
+     - id: json
+       name: JSON
+       content: |
 
-      { "type": "home", "number": "212-555-1234" }
-      { "type": "cell", "number": "646-555-4567" }
-      { "type": "office", "number": "202-555-0182"}
+         When importing data from a **JSON** file, each document must exist on
+         its own line in the file. Do not use commas at the end of lines to
+         separate documents.
 
-   .. raw:: html
+         .. example::
 
-      <br>
+            The following ``.json`` file imports three documents:
+
+            .. code-block:: javascript
+
+               { "type": "home", "number": "212-555-1234" }
+               { "type": "cell", "number": "646-555-4567" }
+               { "type": "office", "number": "202-555-0182"}
+
+            |compass| automatically generates :manual:`ObjectIDs
+            </reference/method/ObjectId/>` for these objects on import since no
+            ObjectIDs were specified in the initial JSON.
+
+     - id: csv
+       name: CSV
+       content: |
+
+         When importing data from a **CSV** file, the first line of the file
+         must be a comma-separated list of your document field names. Subsequent
+         lines in the file must be comma-separated field values in the order
+         corresponding with the field order in the first line.
 
-   |compass| automatically generates :manual:`ObjectIDs
-   </reference/method/ObjectId/>` for these objects on import since no
-   ObjectIDs were specified in the initial JSON.
+         .. example::
 
-To import data into a collection:
+            The following ``.csv`` file imports three documents:
+
+            .. code-block:: none
+
+               name,age,fav_color,pet
+               Jeff,25,green,Bongo
+               Alice,20,purple,Hazel
+               Tim,32,red,Lassie
+
+            |compass| automatically generates :manual:`ObjectIDs
+            </reference/method/ObjectId/>` for these objects on import since no
+            ObjectIDs were specified in the initial CSV file.
```

Comment:
Is compass smart enough to not add null fields from my CSV file to the documents it creates? How do I specify a null field? Can I specify nested content somehow?

Reply:
Still awaiting engineering feedback on this. Can verify during tech review.

Reply:
CC @imlucas, do you know the answer to this?

Reply:
You cannot. I've looked into this further and posted more details to: https://jira.mongodb.org/browse/COMPASS-3129